cwms_radar_api/src/docker/Dockerfile

```diff
@@ -15,6 +15,4 @@ ENV RADAR_POOL_MAX_ACTIVE "10"
 ENV RADAR_POOL_MAX_IDLE "5"
 ENV RADAR_POOL_MIN_IDLE "2"
 
-#EXPOSE 7000
-
-#CMD ["java","-jar","/radar/radar-standalone.jar"]
\ No newline at end of file
+EXPOSE 7000
```

Comment:
Is this for debugging?

Reply:
no that tells the container user hey "this is the port you need to deal with."